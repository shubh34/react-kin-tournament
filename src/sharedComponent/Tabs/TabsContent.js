import './TabsContent.scss';
import React from 'react';
import PropTypes from 'prop-types';

const TabsContent = ({ children }) => (
	<div className="c-tab-content">
		{children}
	</div>
);


TabsContent.propTypes = {
	children: PropTypes.node.isRequired,
};

export default TabsContent;
