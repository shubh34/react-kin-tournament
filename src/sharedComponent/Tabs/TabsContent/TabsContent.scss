

.c-tab-content {
    padding: 24px;
}