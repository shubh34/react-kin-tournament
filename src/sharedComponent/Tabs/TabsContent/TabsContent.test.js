
import React from 'react';
import { shallow } from 'enzyme';
import TabsContent from './TabsContent';

describe('TabsContent', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<TabsContent>TabsContent</TabsContent>);
		expect(wrapper).toMatchSnapshot();
	});
});
