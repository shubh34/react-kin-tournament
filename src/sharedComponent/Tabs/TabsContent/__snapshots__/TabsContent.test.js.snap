// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsContent renders without crashing 1`] = `
<div
  className="c-tab-content"
>
  TabsContent
</div>
`;
